autoscheduler/frontend/src/components/HelpText.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+
+const style = {
+  height: 350,
```

Comment:
Do we have to hard-code the height and width here? Hard-coding is bad for responsive design, since you want your app to be able to respond to different screen sizes, from my 13-inch laptop to a giant desktop monitor. It's generally better to use percents and maxWidth/maxHeight